support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx

```diff
@@ -176,27 +178,36 @@ const ContributionThankYou = ({
       createReferralCodes={campaignSettings && campaignSettings.createReferralCodes}
       campaignCode={campaignSettings && campaignSettings.campaignCode}
     />,
-    shouldShow: true,
+    shouldShow: !isEnvironmentMoment,
   };
   const ausMapAction = {
     component: <ContributionThankYouAusMap />,
     shouldShow: countryId === 'AU',
   };
+  const articleShareAction = {
+    component: <ContributionThankYouArticleShare
+      header="Share Guardian journalism"
+      body="Help extend the reach of our environmental reporting by sharing it with your friends, family and followers."
```

Comment:
Maybe move these into the component? Just to match how the other components do it

Reply:
Done! ✅